src/Hive.IO/Hive.IO/GhDistributors/GhDistSIA.cs

```diff
@@ -27,7 +27,7 @@ protected override void RegisterInputParams(GH_Component.GH_InputParamManager pM
 
         protected override void RegisterOutputParams(GH_Component.GH_OutputParamManager pManager)
         {
-            pManager.AddNumberParameter("Zone Areas", "ZoneAreas", "Floor areas in [m²] of all zones of the building.", GH_ParamAccess.list);
+            pManager.AddNumberParameter("Net Zone Areas", "ZoneAreas", "Net Floor areas in [m²] of all zones of the building.", GH_ParamAccess.list);
```

Comment:
I'm guessing there is no use case for gross floor area? i can't think of any at the moment.

Reply:
I'm using gross area for the text label in the building hizard. I find it more intuitive to expose that number, because it is more likely that you check your hull geometry

Reply:
but for the SIA distributor, I don't think, we need it